web-animations-2/Overview.bs

```diff
@@ -2488,15 +2488,18 @@ Depending on its most recent [=animation trigger state|state=] |state|, the
   The procedure of <dfn>updating animation trigger state</dfn> |state|
   of an [=animation trigger=] |trigger|,
   with a given a flag |did trigger|, and an [=animation trigger type=] |type|,
-  is updated as follows:
+  is as follows:
 
   1. Let |did trigger| be a boolean flag that is initially false.
 
+  1. Let |state| be the current [=animation trigger state|state=]
+    of |trigger| that is initially ''animation trigger state/idle''.
+
   1. Set |state| as follows:
     <dl class=switch>
       : If |trigger|’s [=local time=] is [=unresolved=],
       ::
-        1. Set |state| to ''animation trigger state/idle''.
```

Comment:
Would it make sense to count this as not being inside the trigger range, i.e. switching to state/inverse if we've triggered? It might be a bit odd that you stay in the triggered state when the timeline is inactive.